server/admin_management/api/app/schemas.py

```diff
@@ -69,6 +70,9 @@ class TargetUser(Requester):
     checking "is_new_user()".
     """
     user_id: Optional[int] = None
+    first_name: Optional[Annotated[str, StringConstraints(max_length=50)]] = None
```

Comment:
shall we add a comment there? once we finish the one-time load, the first, last name and email shouldn't be optional anymore?

Reply:
Hi Catherine, this TargetUser schema object can be more flexible since it is only internal that does not return to frontend; and it needs to deal with information gathering for few fields from the search result at separate stage after it is initially initialized. 
Before saving, the ORM model can check for entity fields' correctness, so should be good.
But, right now our db has "first_name", "last_name", "email" as **not** nullable, maybe they also can be nullable as they are from external and cannot always be sure they exist in external system... that's later topic when there is issue...